.circleci/config.yml

```diff
@@ -0,0 +1,174 @@
+version: 2.1
+orbs:
+  git: opuscapita/git@0.0.3
+  github-release: izumin5210/github-release@0.1.1
+
+#  gcp-gcr: circleci/gcp-gcr@0.13.0
+
+
+
+jobs: 
+   build-mac:
+     macos:
+       xcode: 12.0.0
+     environment:
+        CLANG_VERSION: "1:10.0-50~exp1"
+        GO_VERSION: "1.13.7"
+        CMAKE_VERSION: "3.16.3-1ubuntu1"
+        GCC_VERSION: "4:9.3.0-1ubuntu2"
+        GFLAGS_VERSION: "2.2.2-1build1"
+        JSONCPP_VERSION: "1.7.4-3.1ubuntu2"
+        JSONRPCCPP_VERSION: "0.7.0-1build3"
+        SCRYPT_VERSION: "1.21-3"
```

Comment:
Do we need this?

Reply:
honestly testing if iti is actually needed would require time, perhaps it is not.

Reply:
You can just remove and see if everything will work as expected
I assume the vars are already at the docker when executed, and I don't even see where it's used

Reply:
i just removed it, it will work there is not an actual dependency.

Reply:
Did you remove it?